Tools/c-analyzer/cpython/_parser.py

```diff
@@ -47,255 +41,240 @@ def clean_lines(text):
 '''
 
 # XXX Handle these.
-# Tab separated:
-EXCLUDED = clean_lines('''
-# @begin=conf@
-
-# OSX
-Modules/_scproxy.c                # SystemConfiguration/SystemConfiguration.h
-
-# Windows
-Modules/_winapi.c               # windows.h
-Modules/expat/winconfig.h
-Modules/overlapped.c            # winsock.h
-Python/dynload_win.c            # windows.h
-Python/thread_nt.h
-
-# other OS-dependent
-Python/dynload_aix.c            # sys/ldr.h
-Python/dynload_dl.c             # dl.h
-Python/dynload_hpux.c           # dl.h
-Python/emscripten_signal.c
-Python/emscripten_syscalls.c
-Python/thread_pthread.h
-Python/thread_pthread_stubs.h
-
-# only huge constants (safe but parsing is slow)
-Modules/_ssl_data_*.h
-Modules/cjkcodecs/mappings_*.h
-Modules/unicodedata_db.h
-Modules/unicodename_db.h
-Objects/unicodetype_db.h
-
-# generated
-Python/deepfreeze/*.c
-Python/frozen_modules/*.h
-Python/generated_cases.c.h
-Python/executor_cases.c.h
-Python/optimizer_cases.c.h
-# XXX: Throws errors if PY_VERSION_HEX is not mocked out
-Modules/clinic/_testclinic_depr.c.h
-
-# not actually source
-Python/bytecodes.c
-Python/optimizer_bytecodes.c
-
-# mimalloc
-Objects/mimalloc/*.c
-Include/internal/mimalloc/*.h
-Include/internal/mimalloc/mimalloc/*.h
-
-# @end=conf@
-''')
+EXCLUDED = format_table_lines([
+    # OSX
```

Comment:
If updating other things:
```suggestion
    # macOS
```

Reply:
Do whatever makes Ned (and Russell) happy. :)